tests/smartclone_test.go

```diff
@@ -0,0 +1,113 @@
+package tests
+
+import (
+	"fmt"
+	"strings"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"kubevirt.io/containerized-data-importer/pkg/controller"
+	"kubevirt.io/containerized-data-importer/tests/framework"
+	"kubevirt.io/containerized-data-importer/tests/utils"
+
+	cdiv1 "kubevirt.io/containerized-data-importer/pkg/apis/core/v1alpha1"
+)
+
+var _ = Describe("[vendor:cnv-qe@redhat.com][level:component]SmartClone tests", func() {
+
+	var sourcePvc *v1.PersistentVolumeClaim
+
+	fillData := "123456789012345678901234567890123456789012345678901234567890"
+	testFile := utils.DefaultPvcMountPath + "/source.txt"
+	fillCommand := "echo \"" + fillData + "\" >> " + testFile
+
+	f := framework.NewFrameworkOrDie("dv-func-test")
+
+	AfterEach(func() {
+		if sourcePvc != nil {
+			By("[AfterEach] Clean up source PVC")
+			err := f.DeletePVC(sourcePvc)
+			Expect(err).ToNot(HaveOccurred())
+			sourcePvc = nil
+		}
+	})
+
+	Describe("Verify DataVolume Smart Cloning", func() {
+		FIt("succeed creating smart-clone dv", func() {
```

Comment:
Think you forgot to remove "F" from it, so now its just focused on the one test.

Reply:
oops. done. :)